patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1034,7 +1012,7 @@
 +    /**
 +     * Determines if this block can be used as the base of a beacon.
 +     *
-+     * @param world The current world
++     * @param worldObj The current world
```

Comment:
That's the name with the mappings Forge is using. I'll remove these since it would make sense for Forge to update to 1.11 mappings soon.